IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -126,6 +126,7 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
         NSMutableDictionary *newQueryItems = [NSMutableDictionary new];
         NSString *redirectSchemePrefix = [NSString stringWithFormat:@"%@://", s_redirectScheme];
         
+        newQueryItems[MSID_BROKER_IS_PERFORMING_CBA] = @"true";
```

Comment:
has the server side been updated to parse this info?

Reply:
The server won't use this constant. It's just for the Broker so that it is notified to not enable SSO when CBA flow is triggered.